### DISCUSSION TITLE
Custom category

### DISCUSSION THREAD

**Discussion by spaceman66** [EmiyaSyahriel/CrossLauncher#76] [Ideas]
Not sure if exists or not, but is there a way to get apps of choice into a different category. Such as a favorites category or music or video etc. 

If not could be an idea?

**Comment by EmiyaSyahriel** [comment]
You can move apps to another existing categories in the Actions build version. As for creating user-defined custom categories, I've made the issue entry at #78 , It might need a little bit longer to fully implement though.

And also, Music and Video category did exists in the background.

**Comment by Quackdoc** [comment]
It would be nice to have automatic sorting for music, videos, images, and maybe networking for sure. I did wind up installing it on my phone for use with a HDMI cable. it works but there are loads of apps so navigating is a bit tedious

**Reply by EmiyaSyahriel** [reply]
Networking? Do you mean putting browsers and any network-related apps in these category? or is it anything else?

**Reply by Quackdoc** [reply]
correct, Webbrowsers, chatting applications, social media etc
